src/environments/.env.ts

```
@@ -2,8 +2,9 @@
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '250828',
-    'hash': 'b92f79e2'
-  }
+    'version': '251210',
+    'hash': 'g548755ec1'
+  },
+  'allow_switching_backend_instance': true
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The 'allow_switching_backend_instance': true key has incorrect delimiter

(IncorrectDelimiter)